### ISSUE TITLE
crashes 2.51 not working

### ISSUE THREAD

**Issue by oushn** [Whitetigerswt/gtasa_crashfix#63]
crashes 2.51 is not working, it just won't increase fps, 2.4 does it but has that annoying peds sliding bug, fix it. @Whitetigerswt

**Comment by Whitetigerswt** [comment]
Do you know if any of the other features are working besides the fps?

Perhaps it's not loading into the game by the asi loader.

**Comment by oushn** [comment]
thanks for the quick answer, im not home rn so i cant check other features
can you share your asi loader on which crashes work?

**Comment by oushn** [comment]
could you perhaps share your .asi loader, because every other crashes.asi works fine for me? and i see i'm not the only one having problem with this. @Whitetigerswt

**Comment by Whitetigerswt** [comment]
I think which ASI loader you use doesn't matter. I've always used the traditional ASI loader: https://www.gtagarage.com/mods/show.php?id=8321

I think the best way to test it would be go into crashes.cfg, change "sound 1" to "sound 0". If you don't hear anything when you have it set to "sound 0" then you'll know it's loading correctly on your machine. If it's not loading, likely the problem is a missing dependency (but I don't know for sure) 

In 2.51 I changed the compiler to use Visual studio 2017 instead of 2015. 

Make sure you have this installed: https://visualstudio.microsoft.com/vs/older-downloads/ 
![vs2017](https://user-images.githubusercontent.com/1680233/77233419-a74ac280-6b7d-11ea-9385-583d5adc1a2e.png)

(make sure you select x86, even if your PC is x64)

**Comment by oushn** [comment]
I've found solution, after half an hour...
First installed both x86 and x64 C++ 2017 (restarted pc also), didn't work. 
Changed sound parameter to 0 (it worked), and then changed value of parameter ''fps_limit" from 0 to any, literally, and it works(e.g. fps_limit 300).
Thanks for your constructive feedbacks and quick answers, cheers mate! @Whitetigerswt 
If someone gets any more problems with 2.51, lead themto this topic!
